src/main/java/org/springframework/data/tarantool/core/BaseTarantoolTemplate.java

```diff
@@ -356,8 +355,12 @@ protected TupleOperations setNonNullFieldsFromTuple(TarantoolTuple tuple) {
             SingleValueCallResult<TarantoolResult<TarantoolTuple>>>
```

Comment:
@wey1and There is no point then in checking the condition for `void`, because we should not throw an error, if we do not have a metadata, but **returnType** is `void`.